FreeRTOS-Plus/Demo/AWS/Fleet_Provisioning_Windows_Simulator/Fleet_Provisioning_With_CSR_Demo/example_demo_template.json

```diff
@@ -0,0 +1,54 @@
+ {
```

Comment:
This file could be named in a more descriptive way.

Reply:
Addressed in this commit: https://github.com/FreeRTOS/FreeRTOS/pull/724/commits/f1c5cdaa16b75521c57bfec65ed4c759bea63eb6